c7n/resources/ecs.py

```diff
@@ -484,7 +484,8 @@ def augment(self, resources):
         results = []
         client = local_session(self.session_factory).client('ecs')
         for task_def_set in resources:
-            response = client.describe_task_definition(
+            response = super().retry(
```

Comment:
what's the super for, isn't it just self.retry? attributes don't shadow on class defs only methods.

Reply:
Great point! I am not really extending anything here so no need to make use of it.. also just realized it is an attribute/pointer to a func, totally thought it was a method before. Addressed in 12f7784.